pkg/kubelet/util/util_windows.go

```diff
@@ -73,3 +75,13 @@ func NormalizePath(path string) string {
 	}
 	return path
 }
+
+// IsIPv6Supported determines whether the host supports ipv6.
+func IsIPv6Supported() (bool, error) {
```

Comment:
@jsturtevant Could you please review whether this verification method is correct?

Reply:
ping @jsturtevant

Reply:
lgtm, other than the refrence